src/libraries/Microsoft.Extensions.Hosting.Abstractions/src/Environments.cs

```diff
@@ -8,8 +8,17 @@ namespace Microsoft.Extensions.Hosting
     /// </summary>
     public static class Environments
     {
+        /// <summary>
+        /// The development environment can enable features that shouldn't be exposed in production. Because of the performance cost, scope validation and dependency validation only happens in development.
```

Comment:
These field summaries seem more like remarks to me. Here's the guidance for the summary: https://github.com/dotnet/dotnet-api-docs/wiki/Summary%3A-Field.